jenkins/benchmarks/Dockerfile.benchmark

```diff
@@ -0,0 +1,9 @@
+ARG DOCKER_REPO
+FROM ${DOCKER_REPO}/spark:k8s-cicd-spark301
+
+ADD dist/target/rapids-4-spark_2.12-0.3.0-SNAPSHOT.jar /opt/spark/jars/
+ADD integration_tests/target/rapids-4-spark-integration-tests_2.12-0.3.0-SNAPSHOT-jar-with-dependencies.jar /opt/spark/jars/
```

Comment:
I was thinking that it would be better to build new images because the docker images would be cached on each node. If we download jars each time, we will do that for each executor for each query, so will be downloading the jars hundreds of times for each run of the benchmark suite.